src/utils/transcript.ts

```diff
@@ -1,8 +1,7 @@
 import { createHash } from "crypto";
-import { SaveToGHData } from "../../types";
 import config from "@/config/config.json";
 
-export const compareTranscriptBetweenSave = (data: SaveToGHData) => {
+export const compareTranscriptBetweenSave = (data: any) => {
```

Comment:
why is the data type changed to `any`? I saw you removed the `SaveToGHData` type as well. Is there a reason for this? If extra data types were added due to the new data points, we should update the type

Reply:
You can see in the rest of the codebase that this type was removed because it was not needed after the refactoring because the logic now uses existing types.

In this occasion I just switched to `any` while developing and forgot to update. I'll change it.

Reply:
Makes sense